src/config/userSettings.js

```diff
@@ -12,7 +12,13 @@ const userSettings = {
       key: 'finance',
       display: true,
       iconType: iconType.google,
-      type: widgetType.finance,
+      type: widgetType.number,
+      formatter: n => {
```

Comment:
This should be put in a different file with formatters. Shouldn't have re-usable logic in userSettings.